### ISSUE TITLE
TypeError - Friendica\Protocol\ActivityPub\Processor::storeAttachments

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#11799]
- [X ] I have searched open and closed issues for duplicates

### Bug Description
```
Uncaught Exception TypeError: "Argument 2 passed to Friendica\Protocol\ActivityPub\Processor::storeAttachments() must be of the type array, bool given, 
called in .../src/Protocol/ActivityPub/Processor.php on line 219" 
at .../src/Protocol/ActivityPub/Processor.php line 151 
{"exception":"TypeError: Argument 2 passed to Friendica\\Protocol\\ActivityPub\\Processor::storeAttachments() must be of the type array, bool given, called in .../src/Protocol/ActivityPub/Processor.php on line 219 and defined in .../src/Protocol/ActivityPub/Processor.php:151
Stack trace:
#0 .../src/Protocol/ActivityPub/Processor.php(219): Friendica\\Protocol\\ActivityPub\\Processor::storeAttachments()
#1 .../src/Protocol/ActivityPub/Receiver.php(732): Friendica\\Protocol\\ActivityPub\\Processor::updateItem()
#2 .../src/Protocol/ActivityPub/Queue.php(193): Friendica\\Protocol\\ActivityPub\\Receiver::routeActivities()
#3 .../src/Worker/ProcessQueue.php(39): Friendica\\Protocol\\ActivityPub\\Queue::process()
#4 [internal function]: Friendica\\Worker\\ProcessQueue::execute()
#5 .../src/Core/Worker.php(486): call_user_func_array()
#6 .../src/Core/Worker.php(381): Friendica\\Core\\Worker::execFunction()
#7 .../src/Core/Worker.php(106): Friendica\\Core\\Worker::execute()
#8 .../bin/worker.php(86): Friendica\\Core\\Worker::processQueue()
#9 {main}"} - 
{"file":null,"line":null,"function":null,"uid":"d9053d","process_id":189758}
```

### Platform Info

Friendica Version:
2022.09-dev

Friendica Source: 
git

PHP version:
7.4

SQL version:
MariaDB 10.6

**Comment by MrPetovan** [comment]
Paging @Quix0r

**Comment by Quix0r** [comment]
`$item` wasn't fetched and no error handling here.

**Comment by annando** [comment]
We just need to quit earlier, see https://github.com/friendica/friendica/pull/11802

**Comment by nupplaphil** [comment]
Closed with #11802
